src/utils/helpers.js

```diff
@@ -375,6 +375,21 @@ const helpers = {
    */
   plural(qty, singleWord, pluralWord) {
     return qty === 1 ? singleWord : pluralWord;
+  },
+
+  /**
+   * Converts a javascript `Set` instance to a serializable object, which can be stored in redux
+   * @param {Set} set
+   * @returns {Record<string, unknown>}
+   */
+  convertSetToObject(set) {
+    let obj = {};
+
+    for (const item of set) {
+      obj[item] = item;
+    }
+
+    return obj;
```

Comment:
Do we need this?

Since a set will be converted to an object with the same key and value maybe we could just do `Object.fromEntries(set.entries())` and achieve the same result as this function.

The reverse is even simpler, since `new Set(obj.keys())` will create the set.

Reply:
Removed this function on https://github.com/HathorNetwork/hathor-wallet/pull/587/commits/3ee5ac5cfba3a1e7dbe5387109159d06d4a34485 , as the new approach doesn't need it.